src/ui/components/results/verticalresultscomponent.js

```diff
@@ -215,7 +215,8 @@ export default class VerticalResultsComponent extends Component {
       eventType: 'update',
       storageKey: StorageKeys.VERTICAL_RESULTS,
       callback: results => {
-        if (results.searchState === SearchStates.SEARCH_COMPLETE) {
+        if (results.searchState === SearchStates.SEARCH_COMPLETE ||
```

Comment:
Just realizing this now, but would it be possible to call addContainerClass here, instead of in the onMount? that way we can avoid the extra render